src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -52,7 +54,7 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
                     'data-resin-feature': 'annotations',
                 }}
             >
-                {canEdit && (
+                {!shouldDisableModifyMenuItem && canEdit && (
```

Comment:
Does this need to be a variable? Could we just put `false`?

Reply:
I think it can just be false, this is more of a nit so will just merge for now